docs/root/intro/arch_overview/operations/dynamic_configuration.rst

```diff
@@ -46,10 +46,10 @@ CDS
 
 The :ref:`Cluster Discovery Service (CDS) API <config_cluster_manager_cds>` layers on a mechanism by
 which Envoy can discover upstream clusters used during routing. Envoy will gracefully add, update,
-and remove clusters as specified by the API. This API allows implementors to build a topology in
+and remove clusters as specified by the API. This API allows implements to build a topology in
```

Comment:
i think this should be `implementers` or perhaps `implementations`

same below